build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -231,6 +231,8 @@ class SeleniumWebDriverController {
 
 
     await this.driver.manage().window().setRect({
```

Comment:
See if this solves the resize problem?
```
  /**
   * Resize the window to a target client size
   * @param {!Rect} targetClientRect
   */
  async setWindowRect(targetClientRect) {
    const windowRect = await this.driver.manage().window().getRect()
        .catch(e => {this.logError_(e);});

    const clientRect = await this.driver.executeScript(() => {
      return {height: window.innerHeight, width: window.innerWidth};
    });

    // Take window borders into account when calculating the new window size
    const targetWindowRect = {
      height: windowRect.height - clientRect.height + targetClientRect.height,
      width: windowRect.width - clientRect.width + targetClientRect.width,
    };

    const actualWindowRect =
    await this.driver.manage().window().setRect(targetWindowRect)
        .catch(e => {this.logError_(e);});

    if (actualWindowRect.width != targetWindowRect.width ||
      actualWindowRect.height != targetWindowRect.height) {
      this.logError_(new Error(
          'Failed to resize window. ' +
          `Expected: {height: ${targetWindowRect.height}, ` +
          `width: ${targetWindowRect.width}}. ` +
          `Actual: {height: ${actualWindowRect.height}, ` +
          `width: ${actualWindowRect.width}}.`
      ));
    }
  }
```

Reply:
Discussed offline: didn't help

Reply:
Carlos, I thought you saw assertion errors, not resize errors. That would mean that the resize actually worked.

Reply:
To make sure the resize error was actually thrown, it would need this -
```
 logError_(e) {
    throw new Error(e.message);
  }
```

Reply:
There are more timeouts with this method. The tests pass consistently with this PR, barring some Selenium session ID errors